doc/resources-auto.rst

```diff
@@ -0,0 +1,63 @@
+resources_dict: 
+    | type: ``dict``
+    | argument path: ``resources_dict``
+
+    number_node: 
+        | type: ``int``
+        | argument path: ``resources_dict/number_node``
+
+        The number of node need for each `job`
+
+    cpu_per_node: 
+        | type: ``int``
+        | argument path: ``resources_dict/cpu_per_node``
+
+        cpu numbers of each node.
```

Comment:
better clarify that this is the number assigned to each job, not the maximum number available in each node.